freqtrade/tests/testdata/download_backtest_data.py

```diff
@@ -0,0 +1,16 @@
+"""This module download automatically json data of crypto currencies in bittrex"""
+#!/usr/bin/env python3
+
+from urllib.request import urlopen
+
+CURRENCIES = ["ok", "neo", "dash", "etc", "eth", "snt"]
+
+for cur in CURRENCIES:
+	url1 = 'https://bittrex.com/Api/v2.0/pub/market/GetTicks?marketName=BTC-'
+	url2 = '&tickInterval=fiveMin'
+	url = url1+cur+url2
+	x = urlopen(url)
```

Comment:
Please use [requests](http://docs.python-requests.org/en/master/) it is more human readable, e.g.:
```
params = {
    'marketName': 'BTC-',
    'tickInterval': 'fiveMin',
}
res = requests.get('https://...', params=params)
data = res.json()
...
```